.cicd/platforms/pinned/amazon_linux-2-pinned.dockerfile

```diff
@@ -9,15 +9,19 @@ RUN yum update -y && \
     libuuid-devel libtasn1-devel expect socat libseccomp-devel && \
     yum clean all && rm -rf /var/cache/yum
 # install erlang and rabbitmq
-RUN curl -s https://packagecloud.io/install/repositories/rabbitmq/erlang/script.rpm.sh | bash && \
+RUN curl -fsSLO https://packagecloud.io/install/repositories/rabbitmq/erlang/script.rpm.sh && \
```

Comment:
Can bases of those URLs be set in variables, not hardcodied?

Reply:
I don't see what value that has as the base URL(s) aren't reused anywhere within the same docker file.